test/js/scp-api.test.js

```diff
@@ -22,4 +23,38 @@ describe('JavaScript client for SCP REST API', () => {
     const apiData = await fetchFacetFilters('disease', 'tuberculosis')
     expect(apiData.filters).toHaveLength(10)
   })
+
+  // Note: tests that mock global.fetch must be put after tests that don't
+  // mock it.  jest.restoreAllMocks() doesn't clear everything as expected,
+  // nor does anything else.
+  it('includes `Authorization: Bearer` in requests when signed in', done => {
+    // Spy on `fetch()` and its contingent methods like `json()`,
+    // because we want to intercept the outgoing request
+    const mockSuccessResponse = {}
+    const mockJsonPromise = Promise.resolve(mockSuccessResponse)
+    const mockFetchPromise = Promise.resolve({
+      json: () => {mockJsonPromise}
+    })
+    jest.spyOn(global, 'fetch').mockImplementation(() => {
+      mockFetchPromise
+    })
+
+    fetchFacetFilters('disease', 'tuberculosis')
+
+    expect(global.fetch).toHaveBeenCalledWith(
+      expect.anything(),
+      expect.objectContaining({
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+          'Accept': 'application/json',
+          'Authorization': 'Bearer test'
+        }
+      })
+    )
+    process.nextTick(() => {
```

Comment:
I wonder if we'd be better using isolateModules for this type of thing.  I'm not at all used to test environments that don't automatically sandbox tests from each other though, so I don't know what best practices here are

Reply:
Thanks for the pointer!  I haven't used `isolateModules`, but it seems worthwhile.  Noted as something to consider in c8670a3.